docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -186,7 +33,7 @@ then select it in the job wizard.
 |Name |Description |Job |Datafeed
 
 |auth_high_count_logon_events
-|Looks for an unusually large spike in successful authentication events. This can be due to password spraying, user enumeration or brute force activity.
+|looks for an unusually large spike in successful authentication events. This can be due to password spraying, user enumeration or brute force activity.
```

Comment:
I'm not sure what the reason for starting with lowercase is here, and in another row below. Keeping it capitalized is consistent with the other rows in the table.

This edit also adds a serial comma, while we're at it:
```suggestion
|Looks for an unusually large spike in successful authentication events. This can be due to password spraying, user enumeration, or brute force activity.
```

Reply:
Thanks, those were cut and paste from the modules, so that means the same typos will appear in the UI, but alas I didn't catch them there.

Reply:
Created https://github.com/elastic/kibana/pull/132633 to propagate the fixes to the UI text